server_admin/topics/vault.adoc

```diff
@@ -216,3 +216,161 @@ To install and use the above custom provider the configuration would look someth
 The configuration above tells {project_name} to setup the custom Elytron provider and use the key resolver that is created by
 the custom factory.
 endif::[]
+
+=== Sample Configuration
+
+The following is an example of configuring a vault and credential store.  The procedure involves two parts:
+
+* Creating the credential store and a vault, where the credential store and vault passwords are in plain text.
+* Updating the credential store and vault to have the password encrypted by using a mask provided by `elytron-tool.sh`.
+
+In this example, the test target used is an LDAP instance with `BIND DN credential: secret12`. The target is mapped using user federation in the realm `ldaptest`.
+
+==== Configuring the credential store and vault without encryption
+
+You create the credential store and a vault where the credential store and vault passwords are in plain text.
+
+.Prerequisites
+
+* A running LDAP instance has `BIND DN credential: secret12`.  
+
+* The alias uses the format <realm-name>_< key-value> when using the default key resolver. In this case, the instance is running in the realm `ldaptest` and `ldaptest_ldap_secret` is the alias that corresponds to the value `ldap_secret` in that realm.
+
+.Procedure
+
+. Create the credential store.
++
+[source,bash,subs=+attributes]
+----
+[standalone@localhost:9990 /] /subsystem=elytron/credential-store=test-store:add(create=true, location=/home/test/test-store.p12, credential-reference={clear-text=testpwd1!},implementation-properties={keyStoreType=PKCS12})
+----
+
+. Add an alias to the credential store. 
++
+[source,bash,subs=+attributes]
+----
+/subsystem=elytron/credential-store=test-store:add-alias(alias=ldaptest_ldap__secret,secret-value=secret12)
+----
+
+. List the aliases from the credential store.
++
+[source,bash,subs=+attributes]
+----
+keytool -list -keystore ./test-store.p12 -storetype PKCS12 -storepass testpwd1!
```

Comment:
Minor comment: this assumes the keytool command is being run in the /home/test/ folder as that is the full path of the store that we created above. It might be good to change the -keystore param to the full path to avoid any misunderstandings. So I would change it to -keystore /home/test/test-store.p12 in the above command.